.circleci/config.yml

```diff
@@ -161,6 +161,42 @@ jobs:
         command: docker run --env-file=scripts/templates/envvars.env mbed_sdk_py3:latest
           sh -c "source .venv/bin/activate && python scripts/notify.py"
         name: Start the release party!
+  foundation_gen:
+    docker:
+    - image: circleci/python:3.6.3
+    steps:
+    - checkout
+    - run:
+        command: sudo pip install pipenv
+        name: Install pipenv
+    - run:
+        command: pipenv install --pre black
+        name: Install beta version of black for Python formatting
+    - run:
+        command: pipenv install --dev .
+        name: Install SDK with dev dependencies
+    - run:
+        command: pipenv run python scripts/foundation/render_sdk.py api_specifications/public/sdk_foundation_definition.yaml  -vv
+          -p python_definition.yaml -o src/mbed_cloud/sdk
```

Comment:
Might be easier to parse of the whole command is over multiple lines [see](https://github.com/ArmMbedCloud/mbed-cloud-api-contract/blob/master/.circleci/config.yml#L71)
```
command: pipenv run python
    scripts/foundation/render_sdk.py
    api_specifications/public/sdk_foundation_definition.yaml
    -vv
    -p python_definition.yaml
    -o src/mbed_cloud/sdk
```

Reply:
Oh yes, if only this wasn't generated by an evil script which reformats at it's own whim.